lib/routes/routeGET.js

```diff
@@ -17,55 +17,67 @@ export default function routerGET(request, response, log, statsClient) {
                 });
         } else if (request.query.website !== undefined) {
             api.callApiMethod('bucketGetWebsite', request, log,
-                (err, xml) => {
+                (err, xml, corsHeaders) => {
                     statsReport500(err, statsClient);
-                    routesUtils.responseXMLBody(err, xml, response, log);
+                    routesUtils.responseXMLBody(err, xml, response, log,
+                        corsHeaders);
                 });
         } else if (request.query.uploads !== undefined) {
             // List MultipartUploads
             api.callApiMethod('listMultipartUploads', request, log,
-                (err, xml) => {
+                (err, xml, corsHeaders) => {
                     statsReport500(err, statsClient);
-                    return routesUtils.responseXMLBody(err, xml, response, log);
+                    return routesUtils.responseXMLBody(err, xml, response, log,
+                        corsHeaders);
                 });
         } else {
             // GET bucket
-            api.callApiMethod('bucketGet', request, log, (err, xml) => {
-                statsReport500(err, statsClient);
-                return routesUtils.responseXMLBody(err, xml, response, log);
-            });
+            api.callApiMethod('bucketGet', request, log,
+                (err, xml, corsHeaders) => {
+                    statsReport500(err, statsClient);
+                    return routesUtils.responseXMLBody(err, xml, response, log,
+                        corsHeaders);
+                });
         }
     } else {
         if (request.query.acl !== undefined) {
             // GET object ACL
-            api.callApiMethod('objectGetACL', request, log, (err, xml) => {
-                statsReport500(err, statsClient);
-                return routesUtils.responseXMLBody(err, xml, response, log);
-            });
+            api.callApiMethod('objectGetACL', request, log,
+                (err, xml, corsHeaders) => {
+                    statsReport500(err, statsClient);
+                    return routesUtils.responseXMLBody(err, xml, response, log,
+                        corsHeaders);
+                });
             // List parts of an open multipart upload
         } else if (request.query.uploadId !== undefined) {
-            api.callApiMethod('listParts', request, log, (err, xml) => {
-                statsReport500(err, statsClient);
-                return routesUtils.responseXMLBody(err, xml, response, log);
-            });
+            api.callApiMethod('listParts', request, log,
+                (err, xml, corsHeaders) => {
+                    statsReport500(err, statsClient);
+                    return routesUtils.responseXMLBody(err, xml, response, log,
+                        corsHeaders);
+                });
         } else {
             // GET object
             api.callApiMethod('objectGet', request, log, (err, dataGetInfo,
```

Comment:
No worries, there are corsHeaders -- they are added to the existing resMetaHeaders param in the api method: https://github.com/scality/S3/pull/541/files/2a17b1f986738169f32dd80e2f737fdfe921dbc3#diff-dfcd28083d70734f8867f501e8deed28